libpromises/failsafe.cf

```diff
@@ -30,25 +30,44 @@
 # system by fetching policies in case the standard agent run fails.
 ##############################################################################
 
-
-body common control
-{
-      bundlesequence => {
-                          "cfe_internal_checkkeys",
-                          "cfe_internal_update",
-                          "cfe_internal_call_update"
-                        };
-}
-
 body agent control
 {
+      # We can't expect dns to be properly configured everywhere so we
+      # skip the dns check for bootstrap
       skipidentify => "true";
 
       # Bootstrapping can't continue without keys
       abortclasses => { "no_ppkeys_ABORT" };
 }
 
-################################################################################
+###################################################################################################################################
```

Comment:
How many hash marks are too many? :-) I would say anything over the 78th column.